enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestionResult.tsx

```diff
@@ -63,7 +63,7 @@ const ResultView = ({
 
 export const InteractiveQuestionResult = ({
   height,
-  withTitle,
+  withTitle = false,
```

Comment:
since it's a boolean, we don't necessarily have to set the default value to `false`, it's the same type as `withResetButton` so they should behave the same.

Reply:
ok i'll just leave it for now